src/valueTypes/StringValueType/index.js

```diff
@@ -0,0 +1,15 @@
+import { BaseValueType } from '../BaseValueType'
+import ParamStringContainer from './container'
+
+export class StringValueType extends BaseValueType {
+  defaultValue = 'text'
+
+  Component = ParamStringContainer
+
+  doesValueMatch(value) {
+    return typeof value === 'string'
+  }
+
+  compatibleInputs = {
```

Comment:
I get the following error, but only the first time hedron is compiled, if I edit anything in src it works perfectly when it reloads the first time, and sometimes after that upon reloading it freezes when I think this error would occur again.
```
TypeError: Cannot read property 'inputs' of undefined
    at processDevices(MidiInput.js ? 6582 : 68)
    at runCallEffect(/Users/cale.bradbury…nternal / proc.js: 524)
```

I'm wondering if this is due to the fact that this type has no inputs?

Reply:
Also forgot to mention that for the second freeze I get the following in the terminal
```[19645:1123/151459.614385:ERROR:gles2_cmd_decoder.cc(18470)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIUM: invalid mailbox name```

Reply:
Strangely, I'm not getting these errors. Might be a node issue? You shouldn't need the `compatibleInputs` as it's empty in the base class for value type.

Reply:
I think you might be running multiple instances of Hedron and getting these issues. Check your task manager. Sometimes happens while developing!

Reply:
I'll check in tonight and let you know, thanks for the thing to look into!

Reply:
No more errors :) I didn't see those again, I'll keep an eye open for multiple processes in the future though!